src/components/table/_table.scss

```diff
@@ -15,6 +16,19 @@
   > tbody > tr.b-table-details > td {
     border-top: none !important;
   }
+
+  // Busy/empty row styling
+  > tbody > tr {
+    // Dont' make empty row hoverable
+    &.b-table-busy-slot,
+    &.b-table-empty-row {
+      &:hover {
+        color: inherit;
+        background-color: inherit;
```

Comment:
If the empty row has a row variant applied, will inherit take the parents background color rather than the previous defined value?

Reply:
Not sure about that. Didn't even know thats possible.

Reply:
Because it is possible to apply classes to the the empty row.. just wondering if this will override them?

Reply:
I was originally just thinking of checking ththe length of computedItems and removing the table-hover class off of the table element if it is 0

Reply:
That would be another option. When we check for empty and busy, even the better option.

Reply:
Basically we could just do this when applying the hover class

```js
{
  'table-hover': this.hover && this.computedItems.length && !this.computedBusy
}
```